agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -3592,4 +3592,45 @@ bool backhaul_manager::add_ap_he_capabilities(const sRadioInfo &radio_info)
     return true;
 }
 
+const std::string backhaul_manager::freq_to_radio_mac(enum eFreqType freq) const
```

Comment:
(nitpick) although the type is called eFreqType, I think it makes more sense to call the variable `band` instead of `freq`.

Also, AFAIK, our coding style prescribes to drop the `enum`.

Reply:
You are correct, fixed.